src/main/java/com/trolley/trolley/Payment.java

```diff
@@ -35,10 +35,25 @@ public class Payment
     private String methodDisplay;
     private Batch batch;
     private String checkNumber;
-    String withholdingAmount;
-    String withholdingCurrency;
-    String equivalentWithholdingAmount;
-    String equivalentWithholdingCurrency;
```

Comment:
Wasn't sure if these were intentionally package private.
Let me know if that was intentional / feel free to remove